swagger.json

```diff
@@ -0,0 +1,606 @@
+{
```

Comment:
In Hawkular the swagger.json was generated but not committed in the PR.
Travis was the responsible to generate it, transform it, and commit it in the repo
https://github.com/hawkular/hawkular.github.io/tree/swagger

I have no preference, but just wonder if we can re-use something from hawkular repo, or if it is simple to commit the swagger.json with the code.